pkg/kfapp/ossm/templates/control-plane/control-plane-managed.tmpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: {{ .Mesh.Name }}
+  namespace: {{ .Mesh.Namespace }}
+spec:
+  version: v2.4
+  tracing:
+    type: None
+  addons:
+    prometheus:
+      enabled: true
```

Comment:
Curious as to why Prometheus is enabled in this SMCP - in our sample in `enabling-ossm.md` we have it set to [false](https://github.com/maistra/odh-manifests/blob/682f627a4047f9cbb036d2d9d70d17b63b9a9e60/enabling-ossm.md?plain=1#L91).

Reply:
There are no requirements as of now what and if self install is needed. I thought this is helpful as we have metrics from istio collected

Reply:
Gotcha, that makes sense to me :+1: